### PR TITLE
[stable/zookeeper] Update api versions to support k8s 1.16

### PR DESCRIPTION
**Pull request by antongorkovenko**
Signed-off-by: Anton Gorkovenko <gostom@gmail.com>

#### Is this a new chart
no
#### What this PR does / why we need it:
Update api versions to support k8s 1.16
#### Which issue this PR fixes
no
#### Special notes for your reviewer:
no
#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by zanhsieh** [comment]
@antongorkovenko 
Need to resolve conflict on incubator/zookeeper/Chart.yaml

**Comment by antongorkovenko** [comment]
@zanhsieh resolved

File: incubator/zookeeper/templates/statefulset.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
```
Comment (Aisuko) [inline incubator/zookeeper/templates/statefulset.yaml:1]: I'm sorry, but could you please explain why change `v1beta1` to `v1`?

File: incubator/zookeeper/README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 This helm chart provides an implementation of the ZooKeeper [StatefulSet](http://kubernetes.io/docs/concepts/abstractions/controllers/statefulsets/) found in Kubernetes Contrib [Zookeeper StatefulSet](https://github.com/kubernetes/contrib/tree/master/statefulsets/zookeeper).
 
 ## Prerequisites
-* Kubernetes 1.6+
+* Kubernetes 1.10+
```
Comment (Aisuko) [inline incubator/zookeeper/README.md:6]: I'm sorry, What's the mean of this? Statefulset API version is not backward compatible?

**Review by Aisuko — CHANGES_REQUESTED** [review]
Thanks for your PR, There are some questions and expect your reply.

File: incubator/zookeeper/templates/statefulset.yaml (hunk 1 above)
Comment (antongorkovenko) [reply incubator/zookeeper/templates/statefulset.yaml:1]: v1beta has been removed in 1.16: https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

File: incubator/zookeeper/README.md (hunk 2 above)
Comment (antongorkovenko) [reply incubator/zookeeper/README.md:6]: No, new API version is not available in 1.6

**Comment by Aisuko** [comment]
Thanks

**Pull request merged**